src/controllers/carts.js

```diff
@@ -0,0 +1,21 @@
+const cartService = require('../services/cart');
+
+const showCart = async (req, res) => {
+  try {
+    const cart = await cartService.findCartbyUserId(req.body);
```

Comment:
This is ok for now but we should take the user.id from the JWT (like req.user.id) or something like that, if not, any user could see the cart of another user

Reply:
added the comment, same as the one below